src/Sarif.FunctionalTests/ConverterTestData/Semmle/results.csv

```diff
@@ -0,0 +1,80 @@
+"Explicit thread priority","Setting thread priorities to control interactions between threads is not portable, and may not have the desired effect.","warning","Avoid using thread priorities. The effect is unpredictable and not portable.","src/main/java/org/apache/commons/io/FileCleaningTracker.java","commons-io/revision-2016-November-08--14-46-36/src/main/java/org/apache/commons/io/FileCleaningTracker.java","211","13","211","44"
+"Incorrect lazy initialization of a static field","Initializing a static field without synchronization can be problematic in a multi-threaded context.","warning","Incorrect lazy initialization of static field [[""SKIP_BYTE_BUFFER""|""relative://src/main/java/org/apache/commons/io/IOUtils.java:170:6561:16""]]: Missing synchronization.","src/main/java/org/apache/commons/io/IOUtils.java","commons-io/revision-2016-November-08--14-46-36/src/main/java/org/apache/commons/io/IOUtils.java","2718","13","2718","57"
+"Incorrect lazy initialization of a static field","Initializing a static field without synchronization can be problematic in a multi-threaded context.","warning","Incorrect lazy initialization of static field [[""SKIP_CHAR_BUFFER""|""relative://src/main/java/org/apache/commons/io/IOUtils.java:169:6517:16""]]: Missing synchronization.","src/main/java/org/apache/commons/io/IOUtils.java","commons-io/revision-2016-November-08--14-46-36/src/main/java/org/apache/commons/io/IOUtils.java","2793","13","2793","57"
+"Busy wait","Calling 'Thread.sleep' to control thread interaction is less effective than waiting for a notification and may also result in race conditions. Merely synchronizing over shared variables in a loop to control thread interaction may waste system resources and cause performance problems.","warning","Prefer wait/notify or java.util.concurrent to communicate between threads.","src/main/java/org/apache/commons/io/FileUtils.java","commons-io/revision-2016-November-08--14-46-36/src/main/java/org/apache/commons/io/FileUtils.java","1708","21","1708","59"
```

Comment:
>'Thread.sleep' [](start = 21, length = 14)

Note Semmle uses single apostrophes to enquote contents as I also prefer. :P Couldn't resist, sorry.